improve bounties sections (#86)
Joystream/helpdesk: roles/council-members/README.md

This section contains examples with suggestions for the step-by-step workflow some "common" formats for Bounties - namely ["Open"](#open-format), ["Free For All"](#free-for-all-format) and ["Closed"](#close-format) formats. The format should try to optimize for the time, quality, risk and cost, associated with each Bounty.

Note that the target audience for this is not the participants themselves, but rather the Council Members.

#### Definitions of Terms
- Community Repo - this [Github repo](https://github.com/Joystream/community-repo/issues)
- Bounty Issue - meaning a github issue made by `bwhm` or `blrhc` of Jsgenesis in the Community Repo
- Council Member (CM) - a (current) member of the Joystream Council
- Council Secretary - a CM elected by the other CMs to be the "main" representative of the Council towards Jsgenesis
- Bounty Manager (BM) - a Community Member (can be a CM as well) elected by the Council to be responsible for managing all Bounties for some period of time, or "just" this specific Bounty. Defaults to the Council Secretary if no other person is chosen

> - Forum - the on chain [Joystream](https://testnet.joystream.org/#/forum/) forum

Reviewer: ```suggestion
- Forum - the on-chain [Joystream](https://testnet.joystream.org/#/forum/) forum
```

#### Councils Role
As seen in below, the amount of work managing a Bounty will in some cases be substantial. Previously, it was intended that the Council should perform this work as group, but it has become clear that hiring a single (or small group) to act as Bounty Manager(s), either for all open Bounties, or individual ones, is needed for the system to work.

Although the idea of forcing the Council to communicate and co-operate their way through this was noble, it leads to an example of the tragedy of the commons.

As managing the Bounties will be a significant part of their (potential) KPI rewards, they have some excess resources and incentives to hire the "right" people. They can either pay for this out of pocket, request funding for this through spending proposals, reserve some part of the Bounty reward for this purpose or by asking Jsgenesis for designated funding.

> In the end, the Council will still be responsible for the work the BM does, so they should still verify the spending and/or text proposals made, and seek feedback of the BMs performance (eg. responsiveness and communication) from participants.

Reviewer: ```suggestion
In the end, the Council will still be responsible for the work the Bounty Manager does, so they should still verify the spending and/or text proposals made, and seek feedback of the Bounty Manager's performance (eg. responsiveness and communication) from participants.
```

#### General Steps
Jsgenesis creates a Bounty Issue in the Community Repo with a new Bounty. This Bounty Issue is meant for the Council first and foremost, and ideally, the person(s) attempting, or successfully manages, to solve the Bounty should not even *need* to read this.

In this Bounty Issue, Jsgenesis will explain/outline:

> - what the bounties is for

Reviewer: ```suggestion
- what the bounty is for
```

6. If one or more is approved by the Council, the Candidates mark their PR ready for review, and the BM requests a review by Jsgenesis.
8. Jsgenesis grades the work and adds the amount they see applicable to the Fiat Pool.

#### Closed Format
In Bounties that requires significant time and/or other resources to complete, and with only one "winner" in practice, Jsgenesis considers it fair to all parties to have an application process culminating in a single Community Member being assigned the Bounty for some agreed time.

Most `coding` and `resarch` Bounties fits equally well as examples, but [#8 -  "Ledger for Joystream"](https://github.com/Joystream/community-repo/issues/86) seems most suitable due to its extensive scope.

##### Full Workflow
Involved parties, and their responsibilities:
- Jsgenesis
  - initial Bounty Issue
  - final grading
- the Bounty Manager
  - deploying the Bounty
  - advising the Council
  - initial grading
- the Council
  - approving the Bounty
  - voting on spending proposals
- Applicants
  - Community Members applying to be Assigned the Bounty

> - Assigned

Reviewer: ```suggestion
- Assignee
```
